src/NUnitFramework/nunitlite/TextUI.cs

```diff
@@ -40,23 +40,36 @@ public class TextUI
     {
         public ExtendedTextWriter Writer { get; private set; }
 
-        private TextReader _reader;
-        private NUnitLiteOptions _options;
+        private readonly TextReader _reader;
+        private readonly NUnitLiteOptions _options;
 
-        #region Constructors
+        private readonly bool _displayBeforeTest;
+        private readonly bool _displayAfterTest;
+        private readonly bool _displayBeforeOutput;
+
+        #region Constructor
 
         public TextUI(ExtendedTextWriter writer, TextReader reader, NUnitLiteOptions options)
         {
             Writer = writer;
             _reader = reader;
             _options = options;
+
+#if NETSTANDARD1_6 || PORTABLE
+            string labelsOption = options.DisplayTestLabels?.ToUpper() ?? "ON";
+#else
+            string labelsOption = options.DisplayTestLabels?.ToUpper(CultureInfo.InvariantCulture) ?? "ON";
+#endif
+            _displayBeforeTest = labelsOption == "ALL" || labelsOption == "BEFORE";
+            _displayAfterTest = labelsOption == "AFTER";
+            _displayBeforeOutput = _displayBeforeTest || _displayAfterTest || labelsOption == "ON";
         }
 
         #endregion
 
-        #region Public Methods
+#region Public Methods
```

Comment:
I just lined them all up again but maybe we should just acquiesce to keeping them where VS seems to want them. This file probably has too many regions.